src/components/AddVac/index.tsx

```diff
@@ -14,35 +14,35 @@ const AddVac = () => {
     const [email, setEmail] = useState("");
     const handleAddVacancy = async (e: React.FormEvent) => {
         e.preventDefault();
-        try {
-            const response = await axios.post(`${API}/jobs`, {
-                organization_name: nameCompany,
-                price_from: Number(priceFrom),
-                price_to: Number(priceTo),
-                currency: 'currency',
-                position: nameVacancy,
-                city: address,
-                salary: salaryVacancy,
-                phone: phone,
-                type: 'type',
-                id: Math.random(),
-                slug: '',
-                created_at: '',
-                updated_at: '',
-                is_archived: true,
-                gradient: 0,
-                workday: '',
-                organization_icon: '',
-                organization_icon_formats: [null]
-            });
-            if (response.data.success) {
-                console.log(response.data);
+            try {
+                const response = await axios.post(`${API_URL}/jobs`, {
+                    organization_name: nameCompany,
+                    price_from: Number(priceFrom),
+                    price_to: Number(priceTo),
+                    currency: 'currency',
+                    position: nameVacancy,
+                    city: address,
+                    salary: salaryVacancy,
+                    phone: phone,
+                    type: 'type',
+                    id: Math.random(),
+                    slug: '',
+                    created_at: '',
+                    updated_at: '',
+                    is_archived: true,
+                    gradient: 0,
+                    workday: '',
+                    organization_icon: '',
+                    organization_icon_formats: [null]
+                });
+                if (response.data.success) {
+                    console.log(response.data);
+                }
```

Comment:
1. setState'ti boshot.
2. toast menen user'ge feedback ber.
3. navigation kil to vacancy page'ke